pkg/bootloader/grubtemplates/grub_live.cfg

```diff
@@ -0,0 +1,10 @@
+set default=0
+set timeout=5
+set timeout_style=menu
+
+menuentry "{{.DisplayName}}" --class os --unrestricted {
```

Comment:
The `DisplayName` here is the value from the grubBootEntry struct. That's the reason to keep values of grubBootEntry struct public.

Reply:
thanks!